_includes/v23.1/sidebar-data/manage.json

```diff
@@ -161,15 +161,15 @@
               "urls": [
                 "/${VERSION}/backup-validation.html"
               ]
-            },
-            {
-              "title": "Backup and Restore Monitoring",
-              "urls": [
-                "/${VERSION}/backup-and-restore-monitoring.html"
-              ]
             }
           ]
         },
+        {
+          "title": "Backup and Restore Monitoring",
+          "urls": [
+            "/${VERSION}/backup-and-restore-monitoring.html"
+          ]
+        },
```

Comment:
Here I have just moved this page in the nav, so it doesn't imply that monitoring is only available for customer-owned backups